presentation/demosteps.md

```diff
@@ -26,7 +26,7 @@ $ helm search helm101
 
 Deploy: 
 ```bash
-$ helm install helm101/guestbook --name myguestbook --set serviceType=NodePort
+$ helm install helm101/guestbook --name myguestbook --set service.type=NodePort
```

Comment:
Can we keep serviceType=NodePort as a comment that for chart version 0.1.0 we can use it that way?